templates/web/base/front/emergency-message.html

```diff
@@ -1,6 +1,6 @@
 [% emergency_message = c.cobrand.emergency_message %]
 [% IF emergency_message %]
 <div class="emergency-message">
-  [% emergency_message | safe %]
+  [% emergency_message | html_para %]
```

Comment:
The only small issue here is that under our sanitize, we allow them to put their own `p` (and `ul`/`ol`) in the emergency message, and this will then add its own `<p>`s to the mix. The way staff_html_markup does it is to not add paragraphs if there are already any p/ul/ol in the output, assuming they then know what they're doing, this might need something similar, though probably can wait until someone does this in anger and notices.

Reply:
Yeah, perhaps not ideal. I think OK to leave for now and see if it comes up enough to be annoying before fixing it.